### PR TITLE
DEV-1012: Update README to conform to new style guide

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Updated Readme according to new style guide
- Added `.idea/` to .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,10 +231,32 @@ The package is available as open source under the terms of the [MIT License](htt
...
 - **1.1.8** Support `verified_account` and `dwolla_landing` auth flags
-- **1.1.7** Use session over connections for [performance improvement](http://docs.python-requests.org/en/master/user/advanced/#session-objects) ([#8](https://github.com/Dwolla/dwolla-v2-python/pull/8) - Thanks @bfeeser!)
+- **1.1.7** Use session over connections for [performance improvement](http://docs.python-requests.org/en/master/user/advanced/#session-objects) ([#8](https://github.com/Dwolla/dwolla-v2-python/pull/8) - Thanks [@bfeeser](https://github.com/bfeeser)!
 - **1.1.5** Fix file upload bug when using with Python 2 ([#6](https://github.com/Dwolla/dwolla-v2-python/issues/6))
 - **1.1.2** Add `TooManyRequestsError` and `ConflictError`
 - **1.1.1** Add MANIFEST.in
 - **1.1.0** Support per-request headers
 
+## Community
+* If you have any feedback, please reach out to us on [our forums](https://discuss.dwolla.com/) or by [creating a GitHub issue](https://github.com/Dwolla/dwolla-v2-node/issues/new).
```
Comment (ShreyaThapa) [inline README.md:241]:
```suggestion
* If you have any feedback, please reach out to us on [our forums](https://discuss.dwolla.com/) or by [creating a GitHub issue](https://github.com/Dwolla/dwolla-v2-python/issues/new).
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,43 @@
-# DwollaV2
+# Dwolla SDK for Python
 
-Dwolla V2 Python client.
+This repository contains the source code for Dwolla's Python-based SDK, which allows developers to interact with Dwolla's server-side API via a Python API. Any action that can be performed via an HTTP request can be made using this SDK when executed within a server-side environment.
 
-[API Documentation](https://docsv2.dwolla.com)
+### Table of Contents
```
Comment (jcoon97) [inline README.md:5]:
```suggestion
## Table of Contents 
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,43 @@
...
+    - [Setting Headers](#setting-headers)
+    - [Responses](#responses)
+      - [Success](#success)
+      - [Error](#error)
+- [Changelog](#changelog)
+- [Community](#community)
+- [Additional Resources](#additional-resources)
 
-`dwollav2` is available on [PyPi](https://pypi.python.org/pypi/dwollav2), and therefore can be installed automagically via [pip](https://pip.pypa.io/en/stable/installing/).
+---
```
Comment (jcoon97) [inline README.md:21]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,43 @@
...
+
+## Getting Started
+
+### Installation
+
+To begin using this SDK, you will first need to download it to your machine. We use [PyPi](https://pypi.python.org/pypi/dwollav2) to distribute this package from where you can automagically download it via [pip](https://pip.pypa.io/en/stable/installing/).
 
 ```
 pip install dwollav2
```
Comment (jcoon97) [inline README.md:30]:
```suggestion
```shell
$ pip install dwollav2
```

File: README.md [hunk 5]
```diff
@@ -46,53 +63,58 @@ client = dwollav2.Client(
...
-Tokens can be used to make requests to the Dwolla V2 API.
+#### Tokens
 
-### Application tokens
+##### Generating New Access Tokens
 
-Application access tokens are used to authenticate against the API on behalf of a consumer application. Application tokens can be used to access resources in the API that either belong to the application itself (`webhooks`, `events`, `webhook-subscriptions`) or the partner Account that owns the consumer application (`accounts`, `customers`, `funding-sources`, etc.). Application tokens are obtained by using the [`client_credentials`][client_credentials] OAuth grant type:
+Application access tokens are used to authenticate against the API on behalf of an application. Application tokens can be used to access resources in the API that either belong to the application itself (`webhooks`, `events`, `webhook-subscriptions`) or the Dwolla Account that owns the application (`accounts`, `customers`, `funding-sources`, etc.). Application tokens are obtained by using the [`client_credentials`][client_credentials] OAuth grant type:
 
 [client_credentials]: https://tools.ietf.org/html/rfc6749#section-4.4
```
Comment (jcoon97) [inline README.md:72]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -46,53 +63,58 @@ client = dwollav2.Client(
...
-## `Token`
-
-Tokens can be used to make requests to the Dwolla V2 API.
+#### Tokens
 
-### Application tokens
+##### Generating New Access Tokens
 
-Application access tokens are used to authenticate against the API on behalf of a consumer application. Application tokens can be used to access resources in the API that either belong to the application itself (`webhooks`, `events`, `webhook-subscriptions`) or the partner Account that owns the consumer application (`accounts`, `customers`, `funding-sources`, etc.). Application tokens are obtained by using the [`client_credentials`][client_credentials] OAuth grant type:
+Application access tokens are used to authenticate against the API on behalf of an application. Application tokens can be used to access resources in the API that either belong to the application itself (`webhooks`, `events`, `webhook-subscriptions`) or the Dwolla Account that owns the application (`accounts`, `customers`, `funding-sources`, etc.). Application tokens are obtained by using the [`client_credentials`][client_credentials] OAuth grant type:
```
Comment (jcoon97) [inline README.md:70]:
```suggestion
Application access tokens are used to authenticate against the API on behalf of an application. Application tokens can be used to access resources in the API that either belong to the application itself (`webhooks`, `events`, `webhook-subscriptions`) or the Dwolla Account that owns the application (`accounts`, `customers`, `funding-sources`, etc.). Application tokens are obtained by using the [`client_credentials`](https://tools.ietf.org/html/rfc6749#section-4.4) OAuth grant type:
```

File: README.md [hunk 7]
```diff
@@ -46,53 +63,58 @@ client = dwollav2.Client(
...
 
-_Application tokens do not include a `refresh_token`. When an application token expires, generate
-a new one using `client.Auth.client()`._
+_Application access tokens are short-lived: 1 hour. They do not include a `refresh_token`. When it expires, generate a new one using `client.Auth.client()`._
 
-### Initializing pre-existing tokens:
+##### Initializing Pre-Existing Tokens:
 
-`Token`s can be initialized with the following attributes:
+The [Dwolla Sandbox Dashboard](https://dashboard-sandbox.dwolla.com/applications-legacy) allows you to generate `token`s for your application. These tokens can be initialized with the following attributes:
```
Comment (jcoon97) [inline README.md:82]:
```suggestion
The [Dwolla Sandbox Dashboard](https://dashboard-sandbox.dwolla.com/applications-legacy) allows you to generate tokens for your application. A `Token` can be initialized with the following attributes:
```

File: README.md [hunk 8]
```diff
@@ -46,53 +63,58 @@ client = dwollav2.Client(
...
 ```python
 client.Token(access_token = '...',
              expires_in = 123)
 ```
 
-## Requests
+## Making Requests
 
-`Token`s can make requests using the `#get`, `#post`, and `#delete` methods.
+Once you've created a `token`, currently, you can make low-level HTTP requests.
```
Comment (jcoon97) [inline README.md:91]:
```suggestion
Once you've created a `Token`, currently, you can make low-level HTTP requests.
```

File: README.md [hunk 9]
```diff
@@ -221,10 +231,32 @@ The package is available as open source under the terms of the [MIT License](htt
...
-- **1.1.7** Use session over connections for [performance improvement](http://docs.python-requests.org/en/master/user/advanced/#session-objects) ([#8](https://github.com/Dwolla/dwolla-v2-python/pull/8) - Thanks @bfeeser!)
+- **1.1.7** Use session over connections for [performance improvement](http://docs.python-requests.org/en/master/user/advanced/#session-objects) ([#8](https://github.com/Dwolla/dwolla-v2-python/pull/8) - Thanks [@bfeeser](https://github.com/bfeeser)!
 - **1.1.5** Fix file upload bug when using with Python 2 ([#6](https://github.com/Dwolla/dwolla-v2-python/issues/6))
 - **1.1.2** Add `TooManyRequestsError` and `ConflictError`
 - **1.1.1** Add MANIFEST.in
 - **1.1.0** Support per-request headers
 
+## Community
+* If you have any feedback, please reach out to us on [our forums](https://discuss.dwolla.com/) or by [creating a GitHub issue](https://github.com/Dwolla/dwolla-v2-python/issues/new).
+* If you would like to contribute to this library, bug reports and pull requests are welcome on GitHub at https://github.com/Dwolla/dwolla-v2-python.
```
Comment (jcoon97) [inline README.md:242]:
```suggestion
* If you would like to contribute to this library, [bug reports](https://github.com/Dwolla/dwolla-v2-python/issues) and [pull requests](https://github.com/Dwolla/dwolla-v2-python/pulls) are welcome.
```

File: README.md [hunk 10]
```diff
@@ -221,10 +231,32 @@ The package is available as open source under the terms of the [MIT License](htt
...
+
+To learn more about Dwolla and how to integrate our product with your application, please consider visiting the following resources and becoming a member of our community!
+
+* [Dwolla](https://www.dwolla.com/)
+* [Dwolla Developers](https://developers.dwolla.com/)
+* [SDKs and Tools](https://developers.dwolla.com/sdks-tools)
+  * [Dwolla SDK for C#](https://github.com/Dwolla/dwolla-v2-csharp)
+  * [Dwolla SDK for Kotlin](https://github.com/Dwolla/dwolla-v2-kotlin)
+  * [Dwolla SDK for Node](https://github.com/Dwolla/dwolla-v2-node)
+  * [Dwolla SDK for Ruby](https://github.com/Dwolla/dwolla-v2-ruby)
```
Comment (jcoon97) [inline README.md:257]:
```suggestion
  * [Dwolla SDK for PHP](https://github.com/Dwolla/dwolla-swagger-php)
  * [Dwolla SDK for Ruby](https://github.com/Dwolla/dwolla-v2-ruby)
```

File: README.md [hunk 11]
```diff
@@ -221,10 +231,32 @@ The package is available as open source under the terms of the [MIT License](htt
...
+* [Developer Support Forum](https://discuss.dwolla.com/)
+
+---
```
Comment (jcoon97) [inline README.md:260]:
```suggestion
```

**Review by jcoon97 — APPROVED** [review]
Looks good! Left some suggestions for review! :)

**Pull request merged**